bare-metal/apple-silicon/how-to/use-private-networks.mdx

```diff
@@ -0,0 +1,72 @@
+
+For full information about Scaleway Private Networks and VPC, see our [dedicated documentation](/network/vpc/)
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
+
+## How to enable the Private Networks feature
+
+Before you can attach your Mac mini to a Private Network, you must enable the feature. This can be done during [Mac mini creation](/bare-metal/apple-silicon/how-to/create-mac-mini/) or afterwards. These instructions show how to enable the feature after creation of the Mac mini.
+
+1. Click **Apple silicon** in the **Bare Metal** section of the side menu. The Apple silicon splash page displays.
+2. Click the Mac mini you want to enable Private Networks on. The Mac mini's **Overview** page displays.
+3. In the **Private Networks feature** panel, use the <Icon name="toggle" /> toggle button to enable the feature.
+    A pop-up displays, asking you to confirm that you want to enable Private Networks, and showing the estimated cost.
+4. Click **Enable Private Networks**.
+
+Private Networks are enabled, and you are returned to your Mac mini's **Overview** page.
+
+## How to disable the Private Networks feature
+
+You can disable the Private Networks feature at any time after activating it.
+
+1. Click **Apple silicon** in the **Bare Metal** section of the side menu. The Apple silicon splash page displays.
+2. Click the Mac mini you want to disable Private Networks on. The Mac mini's **Overview** page displays.
+3. In the **Private Networks feature** panel, use the <Icon name="toggle" /> toggle button to disable the feature.
+    A pop-up displays, asking you to confirm that you want to disable Private Networks.
+4. Click **Disable Private Networks**.
+
+Private Networks are disabled, and you are returned to your Mac mini's **Overview** page. Any Private Networks previously attached to the Mac mini are now detached. 
+
+## How to attach a Mac mini to a Private Network
+
+1. Click **Apple silicon** in the **Bare Metal** section of the side menu. The Apple silicon splash page displays.
+2. Click the Mac mini you want to attach to a Private Network. The Mac mini's **Overview** page displays.
+3. Click the **Private Networks**tab.
+4. Click **+ Attach to a Private Network**. A pop-up displays.
+5. Select the Private Network you want to attach the Mac mini to.  It will be automatically assigned private IPv4 and IPv6 addresses on this network. The ability to use specific reserved IP addresses for the attachment is planned for the future.
```

Comment:
```suggestion
5. Select the Private Network you want to attach the Mac mini to. It will be automatically assigned private IPv4 and IPv6 addresses on this network. The ability to use specific reserved IP addresses for the attachment is planned for the future.
```